x-pack/solutions/security/plugins/security_solution/server/lib/telemetry/receiver.ts

```diff
@@ -749,6 +754,79 @@ export class TelemetryReceiver implements ITelemetryReceiver {
     return this.esClient().search<RuleSearchResult>(query, { meta: true });
   }
 
+  /**
+   * Find elastic rules SOs which are the rules that have immutable set to true and are of a particular rule type
+   * @returns custom elastic rules SOs with response actions enabled
+   */
+  public async fetchResponseActionsRules() {
+    const query: SearchRequest = {
+      index: `${this.getAlertsIndex()}*`,
+      ignore_unavailable: true,
+      size: 0, // no query results required - only aggregation quantity
+      from: 0,
+      query: {
+        bool: {
+          must: [
+            {
+              term: {
+                'kibana.alert.rule.immutable': false,
+              },
+            },
+            {
+              term: {
+                'kibana.alert.rule.enabled': true,
+              },
+            },
+            {
+              terms: {
+                'kibana.alert.rule.consumer': ['siem', 'securitySolution'],
+              },
+            },
+            {
+              terms: {
+                'kibana.alert.rule.parameters.response_actions.action_type_id': [
+                  '.endpoint',
+                  '.osquery',
+                ],
+              },
+            },
+            {
+              range: {
+                'kibana.alert.rule.updated_at': {
+                  gte: 'now-24h/h',
```

Comment:
the task has the last execution time, do you think it's worth using it instead of 24h? just in case the previous execution failed or didn't run for any reason.